### ISSUE TITLE
Suggestion: Family Tree View

### ISSUE THREAD

**Issue by thelivingphilosophy** [zsviczian/excalibrain#178]
Firstly just want to say that I love Excalibrain and there's so much power in it. There's a couple of thoughts that I had which could make it even more powerful. 

I've noticed that there are two reasons that I want to use Excalibrain: one is for ideation (i.e. coming up with new ideas/making new connections) and the second is context. 

Excalibrain I find wonderful at the latter and much more helpful than the graph view which became unusable once notes reached a certain quantity. But sometimes when I'm in one note in Excalibrain the context escapes me. I can follow a note using my parent/children or next/previous but in attempting to implement an atomic Zettelkasten methodology I quickly get lost.

My idea was to have a sort tree view — this would turbocharge the power of Excalibrain when it comes to the Hierarchy element of LATCH. If it was possible to see parents grandparents all the way up (sort of like Breadcrumbs) or to see the whole sequence of Next Friends and Previous Friends all along then this could be a very powerful way (kind of like how they've laid out Luhmann's notes on the archive (niklas-luhmann-archiv.de) where you can see the sequence of Zettelkasten notes. 

To declutter this view it would be cool (though this would be a whole other feature) to be able to hide (rather than just de-emphasise as we can now (unless I'm missing something)) ontologies. For ideation it's good to see all the friends but for context it's clutter and if this tree view was a thing it would be nice to be able to neatly see just the vertical up/down or horizontal next/prev hierarchies. 
***
### tl;dr 
it would be cool to be able to see just the entire up/down ontology hierarchy (or next/prev) at a glance — from current note all the way up through the parents grandparents etc

**Comment by AMGMNPLK** [comment]
I think all of us want this. It is asked in many posts in different ways. I think this is the key, yes!

**Comment by Dennyyusuf** [comment]
yes this will give us more visibilty

**Comment by elias-code** [comment]
Yes, that was the only reason I just learned excalibrain, only to find out it is very shallow :(
For context: I am currently writing my bachelor thesis and I am getting more and more lost the more I write. I wanted to set up excalibrain to display and browse all my internal cross-refs and visually lay out the thesis this way to get an overview and maybe improve it's structure.
Now that I learned it, I have basically wasted my time to build a fancy navigator.
Don't get me wrong, still great work though, thank you, but the functionality is misleading.
In a real brain, thoughts can jump multiple levels as well, I don't have to think about my father and then his father to think about my great grandfather ...
It would be amazing if the display would be easily customisable. Maybe just hide/unhide single notes or whole branches with one click without needing to first think about a structure which would allow this, like some complicated properties structure :)
